benchmark/Streamly/Benchmark/Data/Array/Foreign/Mut.hs

```diff
@@ -78,7 +82,11 @@ main = do
 
     where
 
-    alloc value = MArray.fromStream (sourceUnfoldrM value 0 :: SerialT IO Int)
+    alloc value =
+        sequence
+        [ MArray.fromStream (sourceUnfoldrM value 0 :: SerialT IO Int)
+        , MArray.fromStream (sourceUnfoldrM value 0 :: SerialT IO Int)
+        ]
```

Comment:
1. A list is not needed, a tuple should be enough.
2. A mutable array is not needed for indices, an immutable array should be enough.